src/Selenium2Library/__init__.py

```diff
@@ -81,7 +101,12 @@ class Selenium2Library(
     | css        | Table Should Contain `|` css=table.my_class `|` text               | Matches by @id or @name attribute |
     | xpath      | Table Should Contain `|` xpath=//table/[@name="my_table"] `|` text | Matches by @id or @name attribute |
 
-    = Timeouts =
+    *Stale and Detached Elements*
```

Comment:
Thought I had pulled that out. It was a placeholder for talking about stale elements [as recommended above](https://github.com/rtomac/robotframework-selenium2library/pull/356#commitcomment-8126221) by @ombre42. Since it was taking too long to compose the doc I was going to add it later...